src/app/test-record/test-record.guard.ts

```diff
@@ -0,0 +1,76 @@
+import { Injectable } from '@angular/core';
```

Comment:
Also we need to break up this guard into PreparersGuard and TestStationGuard, A guard should have single responsibility